apps/api/src/routes/applicationRouter.ts

```diff
@@ -350,6 +350,61 @@ applicationRouter.post(
 	),
 );
 
+applicationRouter.post('/applications/:applicationId/revoke', jsonParser, async (request: Request, response: Response) => {
+	const { applicationId } = request.params;
+
+	if (!applicationId || isNaN(parseInt(applicationId))) {
+		response.status(400).json({
+			message: 'Invalid request. ApplicationId is required and must be a valid number.',
+			errors: 'MissingOrInvalidParameters',
+		});
+		return;
+	}
+
+	try {
+		const applicationIdNum = Number(applicationId);
```

Comment:
I think this will be updated with the use of `withParamSchemaValidation`, but without that, would you consider defining `applicationIdNum` at the start of this block using `parseNum(applicationId || '')`. Its awkward converting it to test and then converting it again for use:

```ts
const applicationIdNum = parseInt(applicationId || '');
	if (isNaN(applicationIdNum)) {
		response.status(400).json({
			message: 'Invalid request. ApplicationId is required and must be a valid number.',
			errors: 'MissingOrInvalidParameters',
		});
		return;
	}
```